### PR TITLE
fix(scss): space before parens inside map

### LINKED ISSUE #2554: SCSS minor space issue
prettier adds a space when formatting this:

```diff
-$map: map-merge($map, ($key: $value));
+$map: map-merge($map,  ($key: $value));
```

Thanks

### PR DESCRIPTION
**Pull request by evilebottnawi**
Fixes #2554

### CODE REVIEW

File: src/language-css/printer-postcss.js [hunk 1]
```diff
@@ -402,7 +402,10 @@ function genericPrint(path, options, print) {
     }
     case "value-paren": {
       if (n.raws.before !== "") {
-        return concat([line, n.value]);
+        const parent = path.getParentNode(2);
+        const isFunction = parent.type === "value-func";
+
+        return concat([isFunction ? "" : line, n.value]);
```
Comment (lydell) [inline src/language-css/printer-postcss.js]: What happens if you have a function call as a parameter? Such as:

```scss
clip: rect(0,  round($card-score-size / 2), 0, 0)
```

Comment (evilebottnawi) [reply src/language-css/printer-postcss.js]: @lydell add to tests

**Comment by evilebottnawi** [comment]
@lydell maybe add more tests with parens?

**Comment by lydell** [comment]
Tests are always welcome, if you feel like it!

**Comment by evilebottnawi** [comment]
Close in favor https://github.com/prettier/prettier/pull/3814

**Pull request closed without merging**